lib/edit-site-page.php

```diff
@@ -132,11 +132,8 @@ function gutenberg_edit_site_init( $hook ) {
 		'maxUploadFileSize' => $max_upload_size,
 	);
 
-	list( $color_palette, ) = (array) get_theme_support( 'editor-color-palette' );
```

Comment:
Do we want to offer the handling of get_theme_support and map it to the new setting like we do for the other theme support flags or is this something that is already handled.